lib/compiler.js

```diff
@@ -68,9 +70,27 @@ exports.compile = function (code, options) {
   }
 
   result.code = magicString.toString();
+
   return result;
 };
 
 exports.transform = function (ast, options) {
   return dynRequire("./transform.js")(ast, options);
 };
+
+function makeUniqueIdQuickly(prefix, source) {
+  const scanRegExp = new RegExp("\\b" + prefix + "(\\d*)\\b", "g");
+  let match, max = -1;
+
+  while ((match = scanRegExp.exec(source))) {
+    max = Math.max(max, +(match[1] || 0));
+  }
+
+  if (max >= 0) {
+    return prefix + (max + 1);
+  }
+
+  return prefix;
+}
+
```

Comment:
It can just be `makeUniqueId` like the exported name for it. Quickly is subjective n all.